java/src/main/java/com/mlt/converter/encodings/GeometryEncoder.java

```diff
@@ -234,18 +242,21 @@ public static Triple<Integer, byte[], Integer> encodeGeometryColumn(
     var mortonDictionaryEncodedSize =
         encodedMortonEncodedDictionaryOffsets.encodedValues.length
             + encodedMortonVertexDictionary.encodedValues.length;
+
     if (plainVertexBufferSize <= dictionaryEncodedSize
         && plainVertexBufferSize <= mortonDictionaryEncodedSize) {
       // TODO: get rid of extra conversion
+      // TODO: refactor to use sorted points vertex buffer
```

Comment:
Curious if you can say more about what would be involved here in "refactor to use sorted points vertex buffer"?

Reply:
The comment is not valid anymore, i forgot to remove, Thanks for the hint!

Reply:
Okay, I can remove the TODO in a followup PR I'm working on now.